app/controllers/notes_controller.rb

```diff
@@ -59,6 +58,10 @@ def confirm_destroy
     authorize @note
   end
 
+  private def redirect_path
```

Comment:
I'd be tempted to make this a method of the model itself, eg `@note.path_to_note`, but that's maybe just personal preference on the thick model / thick controller scale.

Reply:
Yeah, interesting! I think there are lots of ways to do this that possibly come down to preference, as this is Joseph's work I think I will leave this as is, it works for me.